applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc

```diff
@@ -50,11 +50,15 @@ The PostreSQL Database Operator then creates and manages the `Database` resource
 
 .Procedure
 . Switch to the *Developer* perspective and ensure that you are in the appropriate project, for example, `my-petclinic`. In the *Topology* view, hover over the Spring PetClinic sample application to see a dangling arrow on the node.
-. Click and drag the arrow towards the *hippo* database Postgres Cluster to make a binding connection with the Spring PetClinic sample application.
+. Click and drag the arrow towards the *hippo* database Postgres Cluster to make a binding connection with the Spring PetClinic sample application. A *Create Service Binding* modal appears.
```

Comment:
modal > model?

Reply:
```suggestion
. Click and drag the arrow towards the *hippo* database [in the?] Postgres Cluster to make a binding connection with the Spring PetClinic sample application. A *Create Service Binding* [model?] appears.
```

Reply:
@rolfedh Thanks for the suggestion. I'll accept the "in the" part to change. And it is actually "modal" I confirmed it with the SME (@divyanshiGupta and @debsmita1) on this.

Reply:
The word "Modal" appears only once in our OpenShift documentation repo. Modal seems like shorthand for modal dialog box, a term our customer aren't familiar with. Perhaps another term that describes this UI element would be appropriate. Maybe it is a "dialog box"?

Reply:
Oops. The IBM SG guidance has changed:

Write as shown to refer to a secondary window where a user enters information. Do not use "dialog" or
"dialogue box". Use "dialog box" only if the audience is familiar with and expects that terminology. Otherwise,
use "window".

Reply:
@rolfedh Seems fair, then using "dialog box" would be more appropriate here. Thanks!